### PR TITLE
Cert import feature

### PR DESCRIPTION
**Pull request by heidi-holappa**
Django model and serializers updated. Will most likely break stuff as db-schema changes. Will provide instructions to team tomorrow on how to fix issue.

### CODE REVIEW

File: backend/cast_backend/settings.py
Comment (karhelmi) [inline backend/cast_backend/settings.py:1]: Ok. Must ensure backend .env includes line DEBUG=True.

File: backend/restapi/models.py
Comment (karhelmi) [inline backend/restapi/models.py:1]: Great! Certificate and project tables incl. relational tables added and tech table modified as planned.

**Review by karhelmi — CHANGES_REQUESTED** [review]
There are five robot tests that fail. Could you kindly take a look at them? Otherwise looking good!

**Comment by maijams** [comment]
Robot tests fixed by adding "Sleep  1s" commands, since the search page seems to take a bit longer to render than before. One test regarding the skills card disabled by commenting (the card will anyways change with the upcoming features).

**Review by karhelmi — APPROVED** [review]
Good job! Now all the robot tests pass as well.

**Pull request merged**